### PR TITLE
Design rework

### LINKED ISSUE #80: Farbdesign vereinheitlichen
Dinge blau machen
LöschenButtons rot lassen!

### LINKED ISSUE #81: RWD-Design

### PR DESCRIPTION
**Pull request by LucRome**
- closes #80
- closes #81

### CODE REVIEW

File: eCourse_backend/courses/urls.py [hunk 1]
```diff
@@ -22,11 +22,13 @@
     path('delete/<id>', delete_course, name='delte_course'),
     # course
     path('overview/page<int:page>', course_overview, name='course_overview'),
-    # create user iframes
+    # create course iframes
     path('iframes/studentlist/<id>', course_student_list_iframe,
          name='course_studentlist_iframe'),
     # admin: create course
-    path('create/', create_course, name="create_course"),
+    path('create/', create_course_admin, name="create_course"),
```
Comment (phie73) [inline eCourse_backend/courses/urls.py]: revert namechange

File: eCourse_backend/courses/views.py [hunk 2]
```diff
@@ -15,41 +15,53 @@
 
 @xframe_options_exempt
 @login_required
-@permission_required('users.manage_users', raise_exception=True)
-def course_student_list_iframe(request, id, page=1):
+def course_student_list_iframe(request, id):
     # TO:DO Filter students by coursename
```
Comment (phie73) [inline eCourse_backend/courses/views.py]: remove the todo

File: eCourse_backend/courses/views.py [hunk 3]
```diff
@@ -15,41 +15,53 @@
...
+        context = {
+            'lecturer': lecturer,
+            'students': students,
+            'exercise': exercise,
+            'files': files}
+
+        return render(
+            request,
+            'courses/iframes/course_student_list.html',
+            context)
```
Comment (phie73) [inline eCourse_backend/courses/views.py]: files have moved to fileexchange/views.py can be removed here

File: eCourse_backend/courses/views.py [hunk 4]
```diff
@@ -15,41 +15,53 @@
...
+    if (request.user.type == 3):
+        # exercises
+        exercise = Exercise.objects.filter(course_id=id)
+        print(exercise)
+
+        files = dir()
+        for e in exercise:
+            files[e.id] = Submission.objects.filter(
+                (Q(user=request.user.id) | Q(from_lecturer=1)), exercise=e.id)
+
```
Comment (phie73) [inline eCourse_backend/courses/views.py]: see above files have moved

File: eCourse_backend/courses/views.py [hunk 5]
```diff
@@ -113,7 +125,7 @@ def view_course(request, id):
 
 @login_required
 @permission_required('courses.add_course', raise_exception=True)
-def create_course(request):
+def create_course_admin(request):
```
Comment (phie73) [inline eCourse_backend/courses/views.py]: revert the namechange

File: eCourse_backend/eCourse_backend/views.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+from django.shortcuts import render, get_object_or_404
+from django.contrib.auth.decorators import permission_required, login_required
+from users.models import User
```
Comment (phie73) [inline eCourse_backend/eCourse_backend/views.py]: remove unnecessary imports

**Review by phie73 — CHANGES_REQUESTED** [review]
looks good overall, please address the comments

**Review by phie73 — APPROVED** [review]
looks really good. And with file exchange, the file could still be renamed to 'Datei'. See the attached screenshot i just find that the navigation bar is a bit big. But I think that is grumbling on a very high level. and somehow I couldn't find a logout button. but in itself the whole thing can also be merged that way. 
image

**Comment by LucRome** [comment]
I don't know whether you can change the Content of that button, since its provided by the input type="file"

**Comment by LucRome** [comment]
Normaly the logout btn says sign-out and is on the right of the navbar

Was it just missing on a single site?

**Comment by phie73** [comment]
> I don't know whether you can change the Content of that button, since its provided by the input type="file"

oke than it can stay like this

**Comment by phie73** [comment]
> Normaly the logout btn says sign-out and is on the right of the navbar
> 
> Was it just missing on a single site?

i haven't seen it at all idk if i just don't see it or if it really is missing. Can somebody else please have a look?

**Comment by LucRome** [comment]
I normally see it on every page (at least I don't remember one where it missed), could you maybe send a Schreenshot of a Website??
Cause it seems like something we should fix

**Pull request merged**